packages/react-combobox/src/components/ComboButton/renderComboButton.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
+import { getSlots } from '@fluentui/react-utilities';
+import type { ComboButtonState, ComboButtonSlots } from './ComboButton.types';
+
+/**
+ * Render the final JSX of ComboButton
+ */
+export const renderComboButton_unstable = (state: ComboButtonState) => {
+  const { slots, slotProps } = getSlots<ComboButtonSlots>(state);
+
+  return (
+    <slots.root {...slotProps.root}>
+      <slots.content {...slotProps.content}>{slotProps.content.children}</slots.content>
```

Comment:
haha you're right, I was messing around with the render of `slots.content` and forgot to change it back 😅

Reply:
I would suggest that you use the root children as the children for the `content` slot. It's what is done for `MenuItem` and makes for a clearer API, especially if customers want extra dom markup in the content.
```tsx
<ComboButton icon={<Icon />}>
  <span>Some custom content <strong>that is bold</strong></span>
</ComboButton>
```

Which is a lot better than
```tsx
<ComboButton icon={<Icon />} content={<span>Some custom content <strong>that is bold</strong></span>} />
```

The styling is a valid issue since to style the `content` slot only users would need to:

```tsx
<ComboButton icon={<Icon />} content={{ className: styles.content }}>
  <span>Some custom content <strong>that is bold</strong></span>
</ComboButton>
```

In `MenuItem`  we went with the assumption that this styling would not be that common and the trade off was worth the API clarity

Reply:
code in `MenuItem` for reference:
https://github.com/microsoft/fluentui/blob/e2e830792b887733a1fa3db02f1dedb9bf9faa97/packages/react-menu/src/components/MenuItem/useMenuItem.tsx#L59-L62

Reply:
@ling1726 the `content` slot is the primary slot for the `ComboButton` control, so this would already be the way it would be used:

```jsx
<ComboButton icon={<Icon />}>
  <span>Some custom content <strong>that is bold</strong></span>
</ComboButton>
```

ComboButton is a bit different than MenuItem in that any non-style props (e.g. `aria-*` attributes, `disabled`, `readonly`, etc.) should go on the content slot's button, which is the element with `role="combobox"`. The root slot is a `<div>` for styling/layout, so keeping it as the primary slot and just putting its children in the content slot doesn't make as much sense here.